development/tools/cmd/rendertemplates/README.md

```diff
@@ -2,18 +2,18 @@
 
 ## Overview
 
-The Render Templates is a tool that reads the configuration in the [`templates/config.yaml`](../../../../templates/config.yaml) file to generate output files, such as Prow component jobs.
+The Render Templates is a tool that reads the configuration from the [`templates/config.yaml`](../../../../templates/config.yaml) file and data files from [`templates/data`](../../../../templates/data) directory to generate output files, such as Prow component jobs. While `config.yaml` file can hold configuration for output file, you can place such data within data files, which should hold configuration for related output files. Having separate files with grouped data is cleaner and easier to maintain than one huge config file.
 
-The `config.yaml` file specifies the following for the Render Templates:
+The `config.yaml` file and `data files` specifies the following for the Render Templates:
```

Comment:
`data files` should not be formatted as code unless it's a name of a file/directory.
```suggestion
The `config.yaml` file and the data files specify the following for the Render Templates:
```